include/wx/validate.h

```diff
@@ -17,9 +17,44 @@
 
 #include "wx/event.h"
 
+class WXDLLIMPEXP_FWD_CORE wxValidator;
 class WXDLLIMPEXP_FWD_CORE wxWindow;
 class WXDLLIMPEXP_FWD_CORE wxWindowBase;
 
+// ----------------------------------------------------------------------------
+// wxValidationEvent
```

Comment:
I think this should be called `wxValidationErrorEvent`, unless we plan to use it not only for the errors?